demo_nodes_py/demo_nodes_py/parameters/async_param_client.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import threading
+
+from ament_index_python import get_package_share_directory
+import rclpy
+import rclpy.context
+from rclpy.executors import SingleThreadedExecutor
+from rclpy.parameter import Parameter
+from rclpy.parameter import parameter_dict_from_yaml_file
+from rclpy.parameter import parameter_value_to_python
+from rclpy.parameter_client import AsyncParameterClient
+
+
+def main(args=None):
+    target_node_name = 'param_test_target'
+
+    context = rclpy.context.Context()
+    rclpy.init(context=context)
```

Comment:
Correct me if I'm wrong, but I think we can just use the default context to keep things simple,

I.e. just elide passing an explicit context everywhere.

```suggestion
    rclpy.init()
```